3581/3/app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ReviewController < ApplicationController
+  post '/reviews/' do
+    session!
+    @review = Review.new(text: params['text'], grade: params['grade'].to_i,
+                         place_id: params[:place_id], user_id: session[:user_id])
+    @reviews = Review.joins(:user).last(10)
+    if @review.valid?
+      @review.save
+    else
+      flash[:error] = I18n.t(:blank_review)
+    end
+    redirect back
```

Comment:
remove. you need redirect to the restaurant page after success save, otherwise, re-render the page with inputted in input fields attributes